submissions/anon-messenger.md

```diff
@@ -0,0 +1,37 @@
+# Hackathon Submission: Anonynous Messenger
+
+## GitHub handles of Team Members  
+- [@SahilKumarIN](https://github.com/SahilKumarIN)
+
+## Project Title
+Anonymous Messenger
+
+## Project Description    
+Anon-Messenger – a safe space for anonymous communication. Whether you want to create a private room for open discussions, share a secure link for feedback, or communicate without revealing your identity, Anon-Messenger makes it easy and secure. Users can sign up, create rooms, and share the link with anyone. The person with the link can post messages without fear of tracking or identification.
+
+## Inspiration behind the Project  
+The inspiration behind **Anonymous Messenger** was the need for a simple yet powerful tool to help users anonymously message their feelings/confess their feelings. This project inspires from ngl which is an anonymous messagenger platform to send their confession.
+
+## Tech Stack    
+- **React**: For building the frontend.
+- **TailwindCSS**: To style the user interface.
+- **Appwrite**: As a backend service for secure data management, including databases and authentication.
+
+### Usage of Appwrite
+- **Appwrite Databases**: I used Appwrite Databases to store event details such as roomDetails, roomDescription, date, time, and users.
+- **Appwrite Authentication**: Appwrite Authentication was implemented to ensure the create, update, or delete rooms messages.
+
+## Project Repo  
+[Anon messenger GitHub Repo](https://github.com/SahilKumarIN/anon-msngr)
```

Comment:
@SahilKumarIN please make this repo public so that it can be reviewed during judging

Reply:
Actually, I just noticed another repo on your profile. Does it happen to be this one?

```suggestion
[Anon messenger GitHub Repo](https://github.com/SahilKumarIN/anon-messenger)
```